README.md

```diff
@@ -148,17 +148,18 @@ to attempt to force one or more of us to tamper with the software.
 Create a new API key:
 
 ```sh
-$ turnkey gen --name my-test-key
+$ turnkey gen --organization $ORGANIZATION_ID
 {
-    "privateKeyFile": "/Users/rno/.config/turnkey/keys/my-test-key.private",
-    "publicKeyFile": "/Users/rno/.config/turnkey/keys/my-test-key.public"
+   "privateKeyFile": "/Users/andrew/Library/Application Support/turnkey/keys/default.private",
+   "publicKey": "0236f17892a4649d97b2e4a4ad3c22d815e4e77848a0b8e4a5b0956ae4d6be382e",
+   "publicKeyFile": "/Users/andrew/Library/Application Support/turnkey/keys/default.public"
 }
 ```
 
 Make an API request:
 
 ```sh
-$ turnkey request --host coordinator-beta.turnkey.io --path /api/v1/sign --body '{"payload": "hello from TKHQ"}' --key=my-test-key
+$ turnkey request --host coordinator-beta.turnkey.io --path /api/v1/sign --body '{"payload": "hello from TKHQ"}' --key-name=my-test-key
```

Comment:
I'd also remove "my-test-key" from this given we generated a "default" key above.

Maybe it's worth mentioning:
```
Make an API request (this will use the default API key, created above. If you need to sign a request with a different key, use the `--key-name` flag):

...request recipe here...
```